test/testcase.h

```diff
@@ -190,6 +196,47 @@ is_supported_openssl_curve(int nid)
 		EC_GROUP_free(ptr);
 	return ptr ? 1 : 0;
 }
+
+static inline int sw_fallbacks_implemented(int nid)
+{
+	switch (nid) {
+	case NID_X9_62_prime192v1:
+	case NID_secp224r1:
+	case NID_X9_62_prime256v1:
+	case NID_secp384r1:
+	case NID_secp521r1:
+	case NID_brainpoolP160r1:
```

Comment:
Above you have the brainpool curves guarded by `#if OPENSSL_VERSION_NUMBER >= 0x010002000`.  Shouldn't we have the same guard here?

Reply:
Yes, that's true, fixed.